packages/aws-cdk-lib/aws-lambda/test/adot.test.ts

```diff
@@ -21,18 +21,18 @@ describe('ADOT Lambda Layer', () => {
     });
 
     test('is added properly when the region information is available at synthesis time', () => {
-      const layerArn = lambda.AdotLambdaLayerJavaSdkVersion.V1_31_0.layerArn(fn.stack, fn.architecture);
+      const layerArn = lambda.AdotLambdaLayerJavaSdkVersion.V1_32_0.layerArn(fn.stack, fn.architecture);
```

Comment:
```suggestion
      const layerArn = lambda.AdotLambdaLayerJavaSdkVersion.LATEST.layerArn(fn.stack, fn.architecture);
```
Same as with integration, we could use `LATEST` for the unit tests as well.

Reply:
This test is specific to get with the version tag, there is already a test for the latest tag. so the changes are not required here. but the rest of the places are changed to LATEST.